packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -68,18 +80,19 @@ export const OutboundShipmentListViewComponent: FC = () => {
       ],
       'selection',
     ],
-    { onChangeSortBy, sortBy },
+    { onChangeSortBy: () => {}, sortBy },
     [sortBy]
   );
 
   return (
     <>
-      <Toolbar filter={filter} />
+      <Toolbar filter={filter} updateFilter={updateFilterQuery} />
```

Comment:
the filter is being retained - if I filter, then clear the filter then sort the column, the filter reappears in the URL and the list is filtered